easy-ui-react/src/TextField/TextField.module.scss

```diff
@@ -0,0 +1,274 @@
+@use "../styles/common" as *;
```

Comment:
need to check in with design about a few things:

-need to check if the error state / flow aligns with expectations
- i dont think this padding configuration is possible unless the one on the right is referring to the input itself having difference text alignment:

![input](https://github.com/EasyPost/easy-ui/assets/44451175/aa980b10-09a6-4c53-94fe-9d4178ff95af)

- for the icon cases, need to confirm/check padding alignment.
- storybook is reporting an accessibility violation regarding color contrast with the optional text.
- not sure if small input variations can have label text with emphasis